CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ### Maintenance and fixes
 * Updated `from_cmdstan` and `from_numpyro` converter to follow schema convention ([1541](https://github.com/arviz-devs/arviz/pull/1541) and [1525](https://github.com/arviz-devs/arviz/pull/1525))
 * Fix calculation of mode as point estimate ([1552](https://github.com/arviz-devs/arviz/pull/1552))
+* Updated `io_pymc3.py` rename dict to follow schema convention([1555](https://github.com/arviz-devs/arviz/pull/1555))
```

Comment:
Can you add `from_pymc3` to the point above about schema convention? I think it will be more concise and easy to read

Reply:
@OriolAbril Sir, Changed the log. Please let me know if there is anything else to update.Thank you